tap-version-14-specification.md

```diff
@@ -0,0 +1,807 @@
+---
+layout: default
+title: TAP 14 specification
+---
+
+# Status: DRAFT
+
+This is a draft document pending further editing and ratification by no
+fewer than 3 widely used TAP implementations in 3 different language
+communities.
+
+This section will likely be removed prior to ratification.
+
+## Goals of This Specification
+
+* Document the observed behavior of widely used TAP implementations.
+* Add no features that are not already in wide usage across multiple
+  implementations.
+* Explicitly allow what is already allowed, deny what is already denied.
+* Provide updated and clearer guidance for new TAP implementations.
+
+# Name
+
+TAP14 - The Test Anything Protocol v14
+
+## Synopsis
+
+TAP, the Test Anything Protocol, is a simple text-based interface between
+testing modules a test harness. TAP started life as part of the test
+harness for Perl but now has implementations in C/C++, Python, PHP, Perl,
+JavaScript, and probably others by the time you read this.  This document
+describes version 14 of TAP. Go to TAP to read about previous versions.
```

Comment:
```suggestion
describes version 14 of TAP.
```
nit: I think that last sentence is a hangover from when the spec used to be on a wiki.